daisy_workflows/image_build/enterprise_linux/kickstart/rhel_9_byos_arm64.cfg

```diff
@@ -0,0 +1,241 @@
+# rhel-9-options.cfg
+
+### Anaconda installer configuration.
+# Install in text mode.
+text --non-interactive
+harddrive --partition=nvme0n1p2 --dir=/
```

Comment:
`/dev/disk/by-id/google-disk-installer-part2` ? that's what we do in x86 rhel 9 kickstart, and the other kickstarts here

Reply:
Fixed- this was also wrong in the rhel_9_arm64.cfg file (fixed that too).